src/api/ZoweExplorerApiRegister.ts

```diff
@@ -62,6 +63,15 @@ export class ZoweExplorerApiRegister implements ZoweExplorerApi.IApiRegisterClie
         return ZoweExplorerApiRegister.getInstance().getJesApi(profile);
     }
 
+    /**
+     * Static lookup of generic extender API implementation.
+     * @private
```

Comment:
`@private` -> `@static`

Reply:
Good spot. I notice that the other API ones (which I copied as a template) are also wrong. I will raise another issue for the original developers to look at this. Raised https://github.com/zowe/vscode-extension-for-zowe/issues/729